types/chart.js/index.d.ts

```diff
@@ -442,6 +443,7 @@ declare namespace Chart {
         mode?: InteractionMode;
         animationDuration?: number;
         intersect?: boolean;
+        axis?: string;
```

Comment:
How about this instead?

axis?: ’x’ | ’y’ | ’xy’

Since those are the only options the user have.

Reply:
(I’m typing on my phone, hence the crappy formatting)

Reply:
good idea. updated.